ta_services/secure_key_services/include/sks_ta.h

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (c) 2017, Linaro Limited
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#ifndef __SKS_TA_H__
+#define __SKS_TA_H__
+
+#include <sys/types.h>
+#include <stdint.h>
+
+#define TA_SKS_UUID { 0xfd02c9da, 0x306c, 0x48c7, \
+                        { 0xa4, 0x9c, 0xbb, 0xd8, 0x27, 0xae, 0x86, 0xee } }
+
+#define KEYCMD			0
```

Comment:
There seem to be some structure of how the different bits in the command id are assigned. Please add a comment describing the structure and the purpose.